jobs/workflow_test_release.groovy

```diff
@@ -40,17 +50,17 @@ job(name) {
 
   parameters {
     stringParam('WORKFLOW_CLI_SHA', '', "workflow-cli commit SHA")
-    stringParam('WORKFLOW_BRANCH', 'master', "The branch to use for installing the workflow chart.")
-    stringParam('WORKFLOW_E2E_BRANCH', 'master', "The branch to use for installing the workflow-e2e chart.")
-    stringParam('RELEASE', 'rc1', "Release string for resolving workflow-[release](-e2e) charts")
-    stringParam('HELM_REMOTE_REPO', 'https://github.com/deis/charts.git', "The remote repo to use for fetching charts.")
+    stringParam('WORKFLOW_BRANCH', "release-${defaults.workflow.release}", "The branch to use for installing the workflow chart.")
+    stringParam('WORKFLOW_E2E_BRANCH', "release-${defaults.workflow.release}", "The branch to use for installing the workflow-e2e chart.")
+    stringParam('RELEASE', defaults.workflow.release, "Release string for resolving workflow-[release](-e2e) charts")
+    stringParam('HELM_REMOTE_REPO', defaults.helm["remoteRepo"], "The remote repo to use for fetching charts.")
     stringParam('E2E_RUNNER_IMAGE', 'quay.io/deisci/e2e-runner:canary', "The e2e-runner image")
     stringParam('E2E_DIR', '/home/jenkins/workspace/$JOB_NAME/$BUILD_NUMBER', "Directory for storing workspace files")
     stringParam('E2E_DIR_LOGS', '${E2E_DIR}/logs', "Directory for storing logs. This directory is mounted into the e2e-runner container")
   }
 
   triggers {
-    cron('@daily')
+    githubPush()
```

Comment:
Thanks for keeping me honest, @mboersma !  Indeed, strictly speaking, this should ideally have been a separate change.  However, to explain my reasoning:  

I wanted the release test job to trigger off of push events to release branches, rather than arbitrarily running them daily.  The real value of this job is during release time and I don't think there is value in running them arbitrarily during inter-release iterations -- though definitely receptive to feedback on this assumption.